### PR TITLE
Release `v1.5.2`

### PR DESCRIPTION
**Pull request by ramirezfranciscof**

### CODE REVIEW

**Comment by ramirezfranciscof** [comment]
@sphuber should I make any comment in the changelog as to why `v1.5.1` is missing?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## v1.5.1
+## v1.5.2
+
+Note: the patch v1.5.1 was skipped due to a problem when uploading the release to PyPi.
```
Comment (sphuber) [inline CHANGELOG.md:5]:
```suggestion
Note: release `v1.5.1` was skipped due to a problem with the uploaded files to PyPI.
```

**Comment by ramirezfranciscof** [comment]
@sphuber if you agree I think I will just merge this in master (squash it to a single commit) and then merge master into develop.

**Comment by sphuber** [comment]
> @sphuber if you agree I think I will just merge this in master (squash it to a single commit) and then merge master into develop.

To be consistent with other release merges, we shouldn't squash, but create a merge commit

**Review by sphuber — APPROVED** [review]

**Pull request merged**